### ISSUE TITLE
При компиляции в Eclipse, возникает ошибка GyverFilters.cpp:29:1: error: control reaches end of non-void function [-Werror=return-type]

### ISSUE THREAD

**Issue by kang2k10** [AlexGyver/GyverMatrixWiFi#5]
При компиляции в Eclipse, возникает ошибка GyverFilters.cpp:29:1: error: control reaches end of non-void function [-Werror=return-type]

Насколько я понимаю, это из за того, что функция GFilterRA::filteredTime возвращает значение только после того как пройдет заданное время....

**Comment by AlexGyver** [comment]
А компилятор какой?

**Comment by kang2k10** [comment]
gcc c++

**Comment by AlexGyver** [comment]
Avr gcc? Ядро ардуино подключено?

**Comment by kang2k10** [comment]
конечно. остальные скетчи компилируются и работают нормально. косяк именно с этой библиотекой

**Comment by AlexGyver** [comment]
значит где то в коде косяк, который ардуино ИДЕ решает сама

**Comment by kang2k10** [comment]
Прошу прощения, был в командировке. Взял пример GFilterRA, скопировал его в эклипс, и при компиляции ошибка
 error: call of overloaded 'filteredTime(uint16_t)' is ambiguous

**Comment by AndrisJefimovs** [comment]
Тоже появляется ошибка `control reaches end of non-void function [-Werror=return-type]` если стоит `USE_FONTS 1`
